.github/workflows/bsim-tests-publish.yaml

```diff
@@ -25,4 +25,4 @@ jobs:
           commit: ${{ github.event.workflow_run.head_sha }}
           event_file: event/event.json
           event_name: ${{ github.event.workflow_run.event }}
-          files: "bsim-test-results/**/bsim_results.xml"
+          files: "bsim-test-results/bsim_results.xml"
```

Comment:
`**/` is still going to be necessary because the result files will be under `bsim-test-results/bsim_{bluetooth,net}/bsim_results.xml`.